contracts/examples/libraries/Migrate_1_3_0_to_1_2_0.sol

```diff
@@ -1,6 +1,6 @@
 // SPDX-License-Identifier: LGPL-3.0-only
 pragma solidity >=0.7.0 <0.9.0;
-import "./GnosisSafeStorage.sol";
+import "../../libraries/GnosisSafeStorage.sol";
```

Comment:
Why was this adjusted?

Reply:
SignMessageLib uses this library, I thought it makes sense to move it out from examples

Reply:
ohh yeah I missed that change

Reply:
the GnosisSafeStorage contract is not tested because there is nothing to test when it comes to contract interaction :P. But the compiler returns information about the storage layout and it would be cool if we can test that the GnosisSafeStorage and the GnosisSafe have compatible storage layouts.

Reply:
I mean it is audited and it should not block a release. Maybe we should create a follow up task and add tests for it (either you or I in the next 2-3 weeks)